Library/Homebrew/os/linux/elf.rb

```diff
@@ -172,6 +188,16 @@ def needed_libraries_using_readelf(path)
     end
   end
 
+  def patcher
+    @patcher ||= begin
+        Homebrew.setup_gem_environment!
+        require "patchelf"
+        PatchELF::Patcher.new to_s, logging: false
+      rescue LoadError
```

Comment:
can we safely assume it will require every time? And eliminate this bit of code,
I can think of a failing situation when `HOMEBREW_PATCHELF_RB` 
not set during bundle install, but is set at later stage some time. : /

Reply:
Let's go ahead and keep it for now -- part of our incremental rollout will probably be allowing existing users to conditionally enable `HOMEBREW_PATCHELF_RB`, so checking it here and not just at bundle install time is good.